src/placeholder.js

```diff
@@ -23,18 +23,181 @@ class LiveCardModal extends ModalDialog {
+
+                  //if unsuccessful, keep checking other submanifests
+                  if (nRequest[i].readyState == 4 && nRequest[i].status == 404){
+                    console.log('Error: ' + nRequest[i].statusText);
+                    console.log('done... checking again in 15 seconds');
+
+                      //last loop, set timeout to fire the checklive function again in 15 seconds
+                      if(i == xhttpResponseArrayCleaned.length-1){
+
+                        //hide spin
+                        spin[0].style.display='none';
+
+                        setTimeout(function() {
+                          checkLive(src_m3u8);
+                        }, 15000);
+                      }
+
+                  //as soon as ONE submanifest is valid, then check that there are at least 3 segments available before playing (same procedure as before)
+                  }else if (nRequest[i].readyState == 4 && nRequest[i].status == 200){
+                    console.log('Segments found, checking that there are at least 3');
+                    console.log(nRequest[i].responseText);
+
+                    let tsResponse = nRequest[i].responseText,
+                        tsResponseArray,
+                        tsResponseArrayCleaned = new Array();
+
+                    tsResponseArray = tsResponse.split('\n');
+                    console.log('ts array: ');
+                    console.log(tsResponseArray);
+
+                    //loop through and create a cleaned array with the segments only
+                      for (i=0; i < tsResponseArray.length; i++){
+                        if(tsResponseArray[i].charAt(0) != '#' && tsResponseArray[i]){
+                          tsResponseArrayCleaned.push(tsResponseArray[i]);
+                        }
+                      }
+                      console.log('ts clean array: ');
+                      console.log(tsResponseArrayCleaned);
+
+                      console.log('how many segments?');
+                      console.log(tsResponseArrayCleaned.length);
+
+                      //if we have less than 3 segments, keep checking every 15 seconds
+                        if(tsResponseArrayCleaned.length < 3){
+                          console.log('less than 3 segments, check again master m3u8');
+                          setTimeout(function() {
+                            checkLive(src_m3u8);
+                          }, 15000);
+                        }else{
+                          console.log('at least 3 segments available, ready to go!');
+
+                          //remove livecard and stop running
+                          console.log('SUCCESS');
+
+                          //hide spin
+                          spin[0].style.display='none';
+
+                          //remove livecard
+                          player.liveCardModal.close();
+
+                          //player loads current stream
+                          player.load();
```

Comment:
Have used the following and it worked, thanks a lot:

player.one('loadstart', function() {player.play()})
if (player.tech_.hls) {
  player.src(player.tech_.hls.source_);
} else {
  player.load();
}